playbooks/roles/kibana/tasks/main.yml

```diff
@@ -35,5 +35,13 @@
     cp -R * {{ kibana_app_dir }}/htdocs/
 
 - name: copy config
-  template: src=config.js.j2 dest={{ kibana_app_dir }}/htdocs/config.js
+  template: src=config.js.j2 dest={{ kibana_app_dir }}/share/{{ kibana_version }}/config.js
 
+- name: Ensure kibana's nginx site is in place
+  template: src=nginx.conf.j2 dest=/edx/app/nginx/sites-available/kibana mode=0755
```

Comment:
It looks like this isn't checked in yet.
It should probably appear at `playbooks/roles/kibana/templates/nginx.conf.j2`.